docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,256 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
+---
+
+# otelcol.exporter.loadbalancing
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`otelcol.exporter.loadbalancing` accepts traces from other `otelcol` components
+and writes them over the network using the OpenTelemetry Protocol (OTLP) protocol. 
+
+> **NOTE**: `otelcol.exporter.loadbalancing` is a wrapper over the upstream
+> OpenTelemetry Collector `loadbalancing` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.loadbalancing` components can be specified by giving them
+different labels.
+
+`otelcol.exporter.loadbalancing` is especially useful for backends configured with tail-based samplers
+which make a decision based on the view of the full trace.
+
+## Usage
+
+```river
+otelcol.exporter.loadbalancing "LABEL" {
+  resolver {
+    ...
+  }
+  protocol {
+    otlp {
+      client {}
+    }
+  }
+}
+```
+
+## Arguments
+
+`otelcol.exporter.loadbalancing` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`routing_key` | `string` | Routing strategy for load balancing. | `"traceID"` | no
+
+The `routing_key` attribute determines how to route spans across endpoints. Its value could be one of the following:
+* `"service"`: exports spans based on their service name. This is useful when using processors like the span metrics, 
```

Comment:
We don't [yet](https://github.com/grafana/agent/issues/2295) have a span metrics processor :) But I added this comment because it was mentioned in the Collector docs and seems useful